src/app/containers/StoryPromo/index.test.jsx

```diff
@@ -517,4 +521,68 @@ describe('StoryPromo Container', () => {
       });
     });
   });
+
+  describe.only('given the promo is used to render an onward Journey', () => {
```

Comment:
```suggestion
  describe('given the promo is used to render an onward Journey', () => {
```
Merging this would disable test in the CI except this suite!

Reply:
Woops, nice spot. Some tests' tests leftovers 😂